news/2022-31-07-osulazer-audio-design.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "sound design of osu!lazer"
+date: 2022-07-31 00:00:00 +0000
+needs_cleanup: true
+---
+
+![](/wiki/shared/news/2022-07-31-lazer-sound-design/header.png)
+
+Recently I've been very fascinated with the sound design of osu!lazer, so much so that I decided to deep dive it and do a write-up about it!
+
+Sound design is an integral part to how for example, a person experiences opening a can of Coca-Cola or how a car engine sounds, both in real life and in video games. So when I played osu!lazer I was interested if all the UI effects were made from scratch for lazer, and this took me down a rabbit hole. I ended up speaking with nekodex, the person behind the sound design aspect of osu!lazer and wanted to share my findings with you all. 
```

Comment:
quick example of how to write without using "i" or "we" etc. (and a few other changes)

(response to [message on discord](https://discord.com/channels/188630481301012481/218677502141399041/999717945255153735))

it doesn't have to be too dry so using "you" in a few places is ok too

```suggestion
Sound design relates to how any particular sound can make its listener *feel*. The sound itself can be anything from the pop and fizz when opening a can of Coca-Cola, to the rumble of a car engine. In video games, sounds need to match the things players interact with, so for osu! the question is: "What sounds should things like a circle, button, or a toggle have?"

osu!'s new client intends to answer that question with lots of brand new changes, including brand new sound effects to suit the brand new UI. If you are wondering where they come from or how they are made, read along for an interview with [nekodex](https://osu.ppy.sh/users/102), the person behind the sound design of osu!(lazer).
```

Reply:
Thanks, I will take this and re-write the doc to better suit the criteria